### ISSUE TITLE
False positive: Lint unaware of imported binding

### ISSUE THREAD

**Issue by TotalVerb** [tonyhffong/Lint.jl#187]
```julia
julia> lintstr("""
       module Test
       import Base.Iterators: flatten
       export flatten
       end
       """)
1-element Array{Lint.LintMessage,1}:
 none:3 W361 flatten: exporting undefined symbol
```

It is common to re-export imported bindings, say, from submodules.
